src/common/libsubprocess/server.c

```diff
@@ -155,7 +154,8 @@ static void rexec_completion_cb (flux_subprocess_t *p)
         /* no fallback if this fails */
         if (flux_respond_pack (rex->s->h, rex->msg, "{s:s s:i}",
                                "type", "complete",
-                               "rank", rex->s->rank) < 0)
+                               "rank", rex->s->rank)
+                               < 0)
```

Comment:
Looking back at this now, I'm wondering what was left to adjust to make this good enough. The thing I remember trying to work around is the break before trailing `< 0` in if statements, which is why this is a comment on these lines as an example.  Oddly though, looking back at it I kinda don't hate it.  It makes it visually obvious what's being compared here, it can be avoided by wrapping in parentheses, and if we're happy with the rest I'm not sure it's worth blocking on.  @garlick, @grondo, @chu11, thoughts?

Reply:
I really don't like the break before `< 0` which is why I suppose we don't do it in our existing formatting.

Reply:
Odd as it seems, we might also be able to address it with a bit of scripting.  It's a very specific spot, and a vim script of `%g/^\s+\(\!=|==|<|>\) \d.{0,5}$/normal kJ` seems to fix it 100% of the time.  The only downside is that it would mean in a few cases (less than a dozen total I think) we'll end up with lines up to 5 chars longer than the limit set in the clang-format file.  I'm going to throw that into a wrapper script, maybe convert it to sed, and throw up the update.

Reply:
Updated versions of the test files pushed, I made a small mistake so there's a blank line at the beginning of the files, but the trailing comparison operator is taken care of.  Need to figure out how to make it reasonably nice to use this, awk makes the process a bit slow, but does that otherwise take care of it?